### PR TITLE
Fix for OS cluster going to yellow state in dev profile when ISM polices are enabled in VZ CR

### PR DESCRIPTION
**Pull request by anajosep**
Fixes VZ-6538 OS cluster goes to yellow state when the hidden indices created by ISM plugin creates with replica=1 as the dev profile has a single node OS cluster. This causes one secondary node to remain assigned causing the cluster to go into yellow state.

### CODE REVIEW

File: pkg/opensearch/opensearch.go [hunk 1]
```diff
@@ -63,6 +63,10 @@ func (o *OSClient) ConfigureISM(vmi *vmcontrollerv1.VerrazzanoMonitoringInstance
 		}
 
 		opensearchEndpoint := resources.GetOpenSearchHTTPEndpoint(vmi)
+		if err := o.updateDefaultIndexSettings(opensearchEndpoint); err != nil {
```
Comment (anders-swanson) [inline pkg/opensearch/opensearch.go:66]: Should this be called in the ISM function?
Also, any concern with this being called repeatedly?

**Review by anders-swanson — CHANGES_REQUESTED** [review]
Please fix the linter

**Review by anders-swanson — APPROVED** [review]
LGTM

File: pkg/opensearch/opensearch.go (hunk 1 above)
Comment (anajosep) [reply pkg/opensearch/opensearch.go]: Created a separate Go routine based function. There is no issues if this call is made repeatedly. Only this specific section of the index settings is patched and the rest of the settings are persisted.

**Pull request merged**